portal/tasks.py

```diff
@@ -96,6 +95,13 @@ def info():
         current_app.config.get('SERVER_NAME'))
 
 
+@celery.task(bind=True, ignore_result=False, max_retries=3, track_started=True)
```

Comment:
Isn't `ignore_result=False` the default?
```suggestion
@celery.task(bind=True, max_retries=3, track_started=True)
```

Reply:
Did you find the `max_retries=3` necessary?

Reply:
will remove - both came along during hair pulling debugging